pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -337,6 +339,38 @@ func (w *nfdTopologyUpdater) updateNodeResourceTopology(zoneInfo v1alpha2.ZoneLi
 	return nil
 }
 
+func getCPUCoreAttribute(coreKind string) (error, v1alpha2.AttributeInfo) {
+	cpu, err := os.ReadFile(hostpath.SysfsDir.Path("sys/devices/" + coreKind, "cpus"))
+	if os.IsNotExist(err) {
+		klog.ErrorS(err, "error reading cpu entry file", "coreKind", coreKind)
+		return err, v1alpha2.AttributeInfo{}
+	}
```

Comment:
We should **ignore** the error if it's ENOENT. I think we should leave the error handling, or at least logging, to the caller

```suggestion
	if err != nil {
		return v1alpha2.AttributeInfo{}, err
	}
```